src/main/app/src/constants/ActivitiesFilterBar.js

```diff
@@ -55,90 +29,127 @@ const ActivitiesFilterBar = ({activityType, activityTypes, links, updateFiltered
     updateFilteredLinks(links);
   }
 
+  /** This function is to check whether integer entered for number of game players is a valid integer */
+  const checkGameEntryValue = (value) => {
+    let errors = false;
+
+    // Check for negative numbers.
+    if (value.includes("-")) {
+      console.log("invalid number, please enter a positive integer value.");
+      updateGameError(true);
+      errors = true;
+    }
+
+    // Check for non-integers.
+    if (isNaN(value)) {
```

Comment:
You can combine this two if statements together. They have the same logic.

Also checking for NaN before checking for negative number seems to be better. After checking for NaN you can check if the number is < 0. You also need to check if the number is an integer. (I think 3.5 will pass the check you have now).